### PR TITLE
update README and validity

### PR DESCRIPTION
**Pull request by HanniOfHyrule**

### CODE REVIEW

File: src/components/MeasurementsForm.js [hunk 1]
```diff
@@ -8,6 +8,16 @@ const MeasurementsForm = (props) => {
...
 
+  let measurementsCssClass = classes.measurementsInput;
+
+  if (
+    measurementsInput.measurement != null &&
+    measurementsInput.measurement <= 0
+  ) {
+    measurementsCssClass += " " + classes.invalid;
+  }
+  console.log(measurementsCssClass);
```
Comment (domnikl) [inline src/components/MeasurementsForm.js:19]: This console.log can safely be removed.

File: src/components/MeasurementsForm.js [hunk 2]
```diff
@@ -35,8 +49,11 @@ const MeasurementsForm = (props) => {
   const submitHandler = (event) => {
     event.preventDefault();
     props.onAdd(measurementsInput);
+    console.log(measurementsInput);
```
Comment (domnikl) [inline src/components/MeasurementsForm.js:52]: Can also be removed.

File: src/components/Test.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
```
Comment (domnikl) [inline src/components/Test.js:1]: This does not seem to belong here.

**Review by domnikl — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -8,10 +8,11 @@ Housemeter is a tool to archive and evaluate all meter readings in the house.
 
 - you can filter the measurements types with the dropdown
 - you can add new measurements
+- you can filter the mearurementss
```
Comment (domnikl) [inline README.md:11]: double s 😆

Comment (HanniOfHyrule) [reply README.md]: Done!

**Review by domnikl — APPROVED** [review]

**Pull request merged**